Text/CustomEntitySearch/CustomEntitySearch.cs

```diff
@@ -50,10 +50,15 @@ public static class CustomEntitySearch
 
             WebApiSkillResponse response = WebApiSkillHelpers.ProcessRequestRecords(skillName, requestRecords,
                 (inRecord, outRecord) => {
-                    string text = inRecord.Data["text"] as string;
+                    string text = Regex.Escape(inRecord.Data["text"] as string);
```

Comment:
Why do you need to escape this?

Reply:
I am no longer working under the assumption that the characters will only be latin-based

Reply:
Never mind, sorry for this PR this is unnecessary.